testcharms/charm-repo/focal/cockroach/metadata.yaml

```diff
@@ -14,6 +14,9 @@ storage:
 provides:
   db:
     interface: roach
+peers:
+  ring:
+    interface: cockroachdb
```

Comment:
Is this needed?

Reply:
Yes, we do. This was added for the new sidecar test.https://github.com/juju/juju/pull/15225#discussion_r1119607570